### PR TITLE
feat: List farm auction 21 winners

### PR DESCRIPTION
**Pull request by Chef-Yogi**

### CODE REVIEW

File: src/config/constants/farms.ts [hunk 1]
```diff
@@ -42,6 +42,39 @@ const farms: SerializedFarmConfig[] = [
...
   },
   //    * V3 by order of release (some may be out of PID order due to multiplier boost)
+  {
+    pid: 99,
+    lpSymbol: 'XCN-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xF01eD80d46759c0cf6A3e9c66856017d81284962',
+    },
+    token: serializedTokens.xcn,
```
Comment (ChefMomota) [inline src/config/constants/farms.ts:52]: add `isCommunity: true`

File: src/config/constants/farms.ts [hunk 2]
```diff
@@ -42,6 +42,39 @@ const farms: SerializedFarmConfig[] = [
...
+  {
+    pid: 94,
+    v1pid: 525,
+    lpSymbol: 'WZRD-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0xee456d906a38e10680c9d157FFf143390e9681bA',
+    },
+    token: serializedTokens.wzrd,
+    quoteToken: serializedTokens.busd,
```
Comment (chef-jojo) [inline src/config/constants/farms.ts:76]: isCommunity

File: src/config/constants/farms.ts [hunk 3]
```diff
@@ -42,6 +42,39 @@ const farms: SerializedFarmConfig[] = [
...
   //    * V3 by order of release (some may be out of PID order due to multiplier boost)
+  {
+    pid: 99,
+    lpSymbol: 'XCN-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xF01eD80d46759c0cf6A3e9c66856017d81284962',
+    },
+    token: serializedTokens.xcn,
+    quoteToken: serializedTokens.wbnb,
```
Comment (chef-jojo) [inline src/config/constants/farms.ts:53]: isCommunity

File: src/config/constants/farms.ts [hunk 4]
```diff
@@ -42,6 +42,39 @@ const farms: SerializedFarmConfig[] = [
...
+    isCommunity: true,
+  },
+  {
+    pid: 94,
+    v1pid: 525,
+    lpSymbol: 'WZRD-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0xee456d906a38e10680c9d157FFf143390e9681bA',
+    },
```
Comment (ChefMomota) [inline src/config/constants/farms.ts:74]: add `isCommunity: true`

**Comment by ChefMomota** [comment]
lpSymbol: '8PAY-BUSD LP',
lpSymbol: 'HAPPY-BNB',

remove `isCommunity`

**Comment by ChefMomota** [comment]
Add `XCN` image, only upload token png file.

**Comment by Chef-Yogi** [comment]
@ChefMomota
Done

**Review by chefilip — APPROVED** [review]

**Pull request merged**